docs/running-on-yarn.md

```diff
@@ -494,7 +494,7 @@ Similarly, a Hive token will be obtained if Hive is on the classpath, its config
 includes a URI of the metadata store in `"hive.metastore.uris`, and
 `spark.yarn.security.credentials.hive.enabled` is not set to `false`.
 
-If an application needs to interact with other secure HDFS clusters, then
+If an application needs to interact with other secure Hadoop filesystem clusters, then
```

Comment:
remove "clusters" here.  should just say "interact with other secure Hadoop filesystems".